examples/multi-modal-chatbot/application.py

```diff
@@ -18,31 +18,33 @@
 
 
 @action(reads=[], writes=["chat_history", "prompt"])
-def process_prompt(state: State, prompt: str) -> Tuple[dict, State]:
```

Comment:
why remove type annotations completely? I think `-> State` for someone learning helps.